content/reference-rippled.md

```diff
@@ -737,6 +741,181 @@ The response follows the [standard format](#response-formatting), with a success
+            "public_key_hex": "023693F15967AE357D0327974AD46FE3C127113B1110D6044FD41E723689F81CC6",
+            "settle_delay": 86400
+        }],
+        "status": "success"
+    }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The response follows the [standard format](#response-formatting), with a successful result containing the following fields:
+
+| Field      | Type                                       | Description        |
+|:-----------|:-------------------------------------------|:-------------------|
+| `account`  | String                                     | The address of the account from the request; the owner of the payment channels. |
+| `channels` | Array of Channel Objects                   | Payment channels owned by this `account`. |
+| `limit`    | Number                                     | _(May be omitted)_ The limit to how many channel objects were actually returned by this request. |
+| `marker`   | [(Not Specified)](#markers-and-pagination) | _(May be omitted)_ Server-defined value for pagination. Pass this to the next call to resume getting results where this call left off. Omitted when there are no additional pages after this one. |
+
+Each Channel Object has the following fields:
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `account` | String | The owner of the channel, as an [Address][]. |
+| `amount` | String | The amount of [XRP, in drops](#specifying-currency-amounts) allocated to this channel in total. |
+| `balance` | String | The total amount of XRP, in drops, paid out from this channel so far. (You can calculate the amount of XRP remaining in the channel by subtracting `balance` from `amount`.) |
+| `channel_id` | String | A unique ID for this channel, as a 64-character hexadecimal string. This is also the [index of the channel](reference-ledger-format.html#paychannel-index-format) in the ledger's state data. |
+| `destination_account` | String | The destination of the channel, as an [Address][]. Only this account can receive the XRP in the channel while it remains open. |
+| `public_key` | String | _(May be omitted)_ The public key for the payment channel in base58 format, if one was specified at channel creation. Signed claims against this channel must be redeemed with the matching key pair. |
+| `public_key_hex` | String | _(May be omitted)_ The public key for the payment channel in hexadecimal format, if one was specified at channel creation. Signed claims against this channel must be redeemed with the matching key pair. |
+| `settle_delay` | Unsigned Integer | The number of seconds the payment channel must remain open after the owner of the channel requests to close it. |
+| `expiration` | Unsigned Integer | _(May be omitted)_ Time, in seconds since the [Ripple Epoch](#specifying-time), when this channel is set to expire. This expiration date is mutable. If this is before the close time of the most recent validated ledger, the channel is expired. |
```

Comment:
I think the active voice is actively deceptive in this particular case (pun intended). To say "expiration occurs" suggests that an action has taken place. The RCL and the rippled servers don't actually do anything though. They may not even notice that the expiration time has passed.

"Expiration" is more a matter of perspective than an actual occurrence, if that makes sense.

It would be more accurate to say "the RCL rules define a channel in a particular ledger version to be expired if and only if the channel's `Expiration` time is prior to the close time of that particular ledger version's direct parent ledger version." But that's a mouthful.